src/pwr.ps1

```diff
@@ -50,7 +50,7 @@ function Invoke-Airpower {
 				}
 			}
 			'prune' {
-				Invoke-AirpowerPrune
+				Invoke-AirpowerPrune @ArgumentList
```

Comment:
```suggestion
				if ($PSVersionTable.PSVersion.Major -le 5) {
					Invoke-AirpowerPrune @ArgumentList
				} else {
					Invoke-AirpowerPrune $ArgumentList
				}
```

Reply:
I had tried this, but couldn't get it to work properly in one of the PowerShells. But, I'll try again and see if I can get it to work.

Reply:
Yeah, it doesn't work on PowerShell 7.3. It fails to convert an `Object[]` to `Nullable[TimeSpan]`, which makes sense. I'm guessing this works for the other commands because they accept arrays as parameters, but prune only accepts an optional `TimeSpan`.